services/brig/src/Brig/IO/Intra.hs

```diff
@@ -278,9 +278,6 @@ notifyUserDeletionRemotes deleted = do
           case eitherFErr of
             Left fErr -> do
               logFederationError (tDomain uids) fErr
-              -- FUTUTREWORK: Do something better here?
-              -- FUTUREWORK: Write test that this happens
-              throwM $ federationErrorToWai fErr
             Right () -> pure ()
```

Comment:
A pattern like this is encapsulated by `whenLeft :: Applicative m => Either a b -> (a -> m ()) -> m ()`.

Reply:
Ah nice, done.